src/pages/home/live-pricing/_use-live-columns.tsx

```diff
@@ -0,0 +1,37 @@
+import { createColumnHelper } from '@tanstack/react-table'
+import React, { useMemo } from 'react'
+import { TableCell, TableHeaderCell, DailyPercentageCell } from './components/_elements'
+import { TMarketData } from './_types'
+
+const liveMarketColumnHelper = createColumnHelper<TMarketData>()
+
+const useLiveColumns = (marketData: () => void) => {
+    const columns = useMemo(() => {
+        return [
+            liveMarketColumnHelper.accessor('symbol', {
```

Comment:
Create an object of the static content and generate this array of items using "for...in" loop in by a function

Reply:
It's needed to get rid of the duplication of the same array items